docs/source/conf.py

```diff
@@ -305,6 +305,7 @@
     "node_arg_is_weight",
     "return_arg_list",
     # torch.ao.quantization.pt2e.graph_utils
+    "bfs_trace_with_node_process",
```

Comment:
Adding new entries in this file is not ok.
Please make sure that all the new public APIs that you add are properly documented and show up on the website.

Reply:
Thanks for the feedback. Any example I can use for documentation? I noticed that all public APIs under `torch.ao.quantization.pt2e.graph_utils` are in this file and (looks like) there's not any doc for `graph_utils`.

Reply:
If allowed, we can first add `bfs_trace_with_node_process` here, and have another PR for all doc stuff under `graph_utils`.

Reply:
graph_utils shouldn't be public API, I'd recommend to make this private (add `_`)